estate/models/estate_server_actions.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="action_x_estate_property_offer_accept" model="ir.actions.server">
+        <field name="name">Accept Offer</field>
+        <field name="model_id" ref="estate.model_real_estate_property_offer"/>
+        <field name="state">code</field>
+        <field name="code"><![CDATA[
+    for offer in records:
+        offer.write({'x_status': 'accepted'})
+        offer.x_property_id.write({'x_selling_price': offer.x_price, 'x_partner_id': offer.x_partner_id})
+        for other_offer in offer.x_property_id.x_offer_ids:
+            if other_offer != offer:
+                other_offer.write({'x_status': 'refused'})
```

Comment:
Ok, but what if you accept two offers for the same property? It seems like only one will be accepted, but it is kind of magic, no? Shouldn't we take more care (and maybe raise in such cases)?